contrib/awssqs/samples/awssqs-source.yaml

```diff
@@ -1,11 +1,10 @@
 # Replace the following before applying this file:
 #   QUEUE_URL: Replace with the AWS SQS queue.
```

Comment:
is this right?

Reply:
Should be, the URL points at the SQS you'd be pulling messages from.